cnf-certification-test/accesscontrol/suite.go

```diff
@@ -553,7 +554,7 @@ func testAutomountServiceToken(env *provider.TestEnvironment) {
 	failedPods := []string{}
 	for _, put := range env.Pods {
 		ginkgo.By(fmt.Sprintf("check the existence of pod service account %s (ns= %s )", put.Namespace, put.Name))
-		if put.Spec.ServiceAccountName == "" {
+		if put.Spec.ServiceAccountName == defaultServiceAccount {
 			tnf.ClaimFilePrintf("Pod %s has been found with an empty service account name.", put.Name)
```

Comment:
This should probably be changed to:

```suggestion
			tnf.ClaimFilePrintf("Pod %s has been found with a default service account", put.Name)
```

Reply:
Updated. Thanks!